docs/src/app/components/pages/components/lists.jsx

```diff
@@ -34,11 +36,40 @@ const { Colors } = Styles;
 const Code = require('lists-code');
 const CodeExample = require('../../code-example/code-example');
 const CodeBlock = require('../../code-example/code-block');
+let SelectableList = SelectableContainerEnhance(List);
+
+function wrapState(ComposedComponent) {
```

Comment:
Yeah, it is avoidable, but it makes the `<ListsPage>` really slow. There is a noticable lag if you click on an item of the `<SelectableList>`. See the performance-Section of my previous comment.